apis/apiextensions/v2alpha1/mrd_policy_types.go

```diff
@@ -25,6 +28,12 @@ import (
 // ActivationPolicy matches on MRD names with wildcard prefix support.
 type ActivationPolicy string
 
+// Match reports whether name matches the activation policy pattern.
+func (a ActivationPolicy) Match(name string) bool {
+	match, _ := filepath.Match(string(a), name)
+	return match
+}
+
 // ManagedResourceActivationPolicySpec specifies the desired activation state of ManagedResourceDefinitions.
 type ManagedResourceActivationPolicySpec struct {
 	// Activations is an array of MRD names to activate. Supports wildcard
```

Comment:
GitHub isn't letting me comment on line +41, but should `+required` be `// +kubebuilder:validation:MinItems=1`?

Reply:
fixed.